README.md

```diff
@@ -86,301 +77,121 @@ database.
-certificates $CA_CRT should be present in /etc/openvswitch/k8s-ca.crt or
-stored in the OVSDB.  (Please note that if no protocol is specified in
-external_ids:k8s-api-server, insecure http connection will be used.)
-
-```
-ovs-vsctl set Open_vSwitch . \
-  external_ids:k8s-api-server="https://$K8S_API_SERVER_IP" \
-  external_ids:k8s-ca-certificate="$CA_CRT" \
-  external_ids:k8s-api-token="$API_TOKEN"
-```
-
-Clone this repository and install the executables.
-
-```
-git clone https://github.com/openvswitch/ovn-kubernetes
-cd ovn-kubernetes
-pip install .
-```
-
-On the minion node, with a unique node name of $NODE_NAME, for a cluster wide
-private address range of $CLUSTER_IP_SUBNET and a smaller subnet for the
-logical switch created in the minion node of $MINION_SWITCH_SUBNET, run:
-
-```
-ovn-k8s-overlay minion-init \
-  --cluster-ip-subnet="$CLUSTER_IP_SUBNET" \
-  --minion-switch-subnet="$MINION_SWITCH_SUBNET" \
-  --node-name="$NODE_NAME"
-```
-
-Note: $NODE_NAME should be the same as the one used by kubelet.
-kubelet by default uses the hostname.  kubelet allows this name to be be
-overridden with --hostname-override.
-
-An example is:
-
-```
-ovn-k8s-overlay minion-init \
-  --cluster-ip-subnet="192.168.0.0/16" \
-  --minion-switch-subnet="192.168.2.0/24" \
-  --node-name="kube-minion1"
-```
-
-Note on Windows: ```ovn-k8s-overlay``` cannot be invoked directly and has to
-be called via the python interpreter. The following is an example ran from an
-elevated powershell prompt.
-```
-python (Get-Command ovn-k8s-overlay).Source minion-init \
-  --cluster-ip-subnet="$CLUSTER_IP_SUBNET" \
-  --minion-switch-subnet="$MINION_SWITCH_SUBNET" \
-  --node-name="$NODE_NAME"
-```
-
-An example is:
-```
-python (Get-Command ovn-k8s-overlay).Source minion-init \
-  --cluster-ip-subnet="192.168.0.0/16" \
-  --minion-switch-subnet="192.168.2.0/24" \
-  --node-name="kube-minwin1"
-```
-
-### k8s gateway node initialization
-
-Gateway nodes are needed for North-South connectivity.  OVN has support
-for multiple gateway nodes.
-
-On any minions (or separate nodes, which would be the case for a DPDK
-based OVN gateway), you need to initialize the gateway node.
-
-Set the k8s API server address in the Open vSwitch database for the
-initialization scripts (and later daemons) to pick from.
-
-```
-ovs-vsctl set Open_vSwitch . \
-  external_ids:k8s-api-server="$K8S_API_SERVER_IP:8080"
-```
+If you want to use SSL instead of TCP for OVN databases, please read
+[INSTALL.SSL.md].
 
-Clone this repository and install the executables.
+### Minion node initialization.
 
-```
-git clone https://github.com/openvswitch/ovn-kubernetes
-cd ovn-kubernetes
-pip install .
-```
+On each host, you will need to run the following command once.
 
-You have two options with respect to choosing the interface via which
-North-South connectivity happens.
+The below command expects the user to provide
+* A cluster wide private address range of $CLUSTER_IP_SUBNET
```

Comment:
Can $CLUSTER_IP_SUBNET on node be different with master? If not, is it worth mentioning $CLUSTER_IP_SUBNET should be the same as master?

Reply:
which brings the idea that maybe the master should store it and the node just learns it, through ovsdb or apiserver.